span.go

```diff
@@ -58,3 +62,30 @@ type Span interface {
 	// TraceContext.
 	AddToGoContext(goCtx context.Context) (Span, context.Context)
 }
+
+// See Span.Log(). Every LogData instance should specify at least one of Event
+// and/or Payload.
+type LogData struct {
+	// The timestamp of the log record; if set to the default value (the unix
+	// epoch), implementations should use time.Now() implicitly.
+	Timestamp time.Time
+
+	// Event (if non-empty) should be the stable name of some notable moment in
+	// the lifetime of a Span. For instance, a Span representing a browser page
+	// load might add an Event for each of the Performance.timing moments
+	// here: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceTiming
+	//
+	// While it is not a formal requirement, Event strings will be most useful
+	// if they are *not* unique; rather, tracing systems should be able to use
+	// them to understand how two similar Spans relate from an internal timing
+	// perspective.
+	Event string
```

Comment:
This is an RFC, per https://github.com/opentracing/opentracing.github.io/issues/30#issuecomment-173756516

The idea is to focus on the propagation API and the non-controversial parts of the logging API, get some experience integrating into applications and tracers, then add human-readable logging afterwards. I.e., I would see this as a strategic deferral, not a removal.